gui/default/syncthing/core/connectivityStatusModalView.html

```diff
@@ -0,0 +1,54 @@
+<modal id="connectivity-status" status="{{connectivityStatusParams.status}}" icon="fas fa-fw {{connectivityStatusParams.type == 'listeners' ? 'fa-sitemap' : 'fa-map-signs'}}" heading="{{connectivityStatusParams.heading}}" large="no" closeable="yes">
+  <div class="modal-body" ng-switch="connectivityStatusParams.type">
+    <div ng-switch-when="listeners">
+      <div ng-if="listenersRunning.length > 0">
+	<p translate>
+	  The following network ports and protocols are available for other devices to connect to this one:
+	</p>
+	<ul>
+	  <li ng-repeat="listener in listenersRunning">{{listener}}</li>
+	</ul>
+      </div>
+      <div ng-if="listenersFailed.length > 0">
+	<p translate>
+	  Some network ports or protocols could not be enabled to accept connections:
+	</p>
+	<ul>
+	  <li ng-repeat="listener in listenersFailed">{{listener}}</li>
+	</ul>
```

Comment:
Indentation.

Reply:
Same below, and in one place it's 4 instead of 2 spaces. Easier to just let some formatter do its magic instead of commenting on each of them :)

Reply:
Sorry, that's just my emacs HTML mode defaulting to mixed tabs and spaces.  Need to get that under control soon, as I often forget to `untabify`.